addons/payment/__init__.py

```diff
@@ -7,6 +7,16 @@
 from . import controllers
 from . import wizards
 
+
 def uninstall_hook(cr, registry):
     env = api.Environment(cr, SUPERUSER_ID, {})
     env['payment.transaction'].search([]).unlink()
+
+
+def reset_payment_provider(cr, registry, provider):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    acquirers = env['payment.acquirer'].search([('provider', '=', provider)])
+    acquirers.write({
+        'view_template_id': acquirers._get_default_view_template_id().id,
```

Comment:
Since it relies solely on data, there is still a (very smal) risk that it crashes.

https://github.com/odoo/odoo/blob/5aaacb825d0c408bc11c4fbb386e1d365a68594b/addons/payment/models/payment_acquirer.py#L67

On the other hand, I don't see another solution since the field is required.